packages/core/test/todo.md

```diff
@@ -0,0 +1,2022 @@
+    }, {
+        key: 'BACKSPACE',
+    }, {
+        key: 'BACKSPACE',
+    }, {
+        key: 'BACKSPACE',
+    }],
+    test: "<p>abc&nbsp;◆def</p><ul><li><p>dom not to edit</p></li></ul>",
+    // todo: handle remove nbsp (not handled because no change triggered in "abc&nbsp;" so no rules applied)
+},
+{
+    name: "in li > p: BACKSPACE after single character",
+    content: "<ul><li><p>a◆</p></li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ul><li><p>▶<br/>◀</p></li></ul>",
+},
+{
+    name: "in li > p: BACKSPACE -> 'a' after single character",
+    content: "<ul><li><p>a◆</p></li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }, {
+        key: 'a',
+    }],
+    test: "<ul><li><p>a◆</p></li></ul>",
+},
+{
+    name: "in li > p > b: BACKSPACE on whole b",
+    content: "<ul><li><p>a<b>▶bcd◀</b>e</p></li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ul><li><p>a◆e</p></li></ul>",
+},
+{
+    name: "in li (two empty li's before): BACKSPACE at start",
+    content: "<ul><li><br/></li><li><br/></li><li>◆kl</li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ul><li><br/></li><li>◆kl</li></ul>",
+},
+{
+    name: "in ul > li (ol > li before): BACKSPACE at end",
+    content: "<ol><li>a</li></ol><ul><li>◆b</li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ol><li>a◆b</li></ol>",
+},
+{
+    name: "in ul > li > p (ol > li before): BACKSPACE at end",
+    content: "<ol><li>a</li></ol><ul><li><p>◆b</p></li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ol><li>a◆b</li></ol>",
+},
+{
+    name: "in ul > li (ol > li > p before): BACKSPACE at end",
+    content: "<ol><li><p>a</p></li></ol><ul><li>◆b</li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ol><li><p>a◆b</p></li></ol>",
+},
+{
+    name: "in ul > li > p (ol > li > p before): BACKSPACE at end",
+    content: "<ol><li><p>a</p></li></ol><ul><li><p>◆b</p></li></ul>",
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: "<ol><li><p>a◆b</p></li></ol>",
+},
+{
+    name: "in p (ul before): BACKSPACE at start",
+    content: '<ul><li><p>node to merge with</p></li></ul><p>◆node to merge</p>',
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: '<ul><li><p>node to merge with◆node to merge</p></li></ul>',
+},
+{
+    name: "in p > b (ul before, i after): BACKSPACE at start",
+    content: '<ul><li><p>node to merge with</p></li></ul><p><b>◆node</b><i>to merge</i></p>',
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: '<ul><li><p>node to merge with<b>◆node</b><i>to merge</i></p></li></ul>',
+},
+{
+    name: "in p > b (ul > i before, i after): BACKSPACE at start",
+    content: '<ul><li><p><i>node to merge with</i></p></li></ul><p><b>◆node</b><i>to merge</i></p>',
+    steps: [{
+        key: 'BACKSPACE',
+    }],
+    test: '<ul><li><p><i>node to merge with</i><b>◆node</b><i>to merge</i></p></li></ul>',
+},
+```
```

Comment:
Syntax highlighting is messed up in this part of the file. Is this a github issue or is there something wrong ?